otter/grade/containers.py

```diff
@@ -164,22 +144,20 @@ def grade_assignments(submission_path, image="ucbdsinfra/otter-grader", verbose=
         nb_basename = os.path.basename(submission_path)
         nb_name = os.path.splitext(nb_basename)[0]
 
-        volumes = {
-            f"{temp_subm_path}": {'bind': f"/autograder/submission/{nb_basename}"},
-            f"{results_path}": {'bind': "/autograder/results/results.pkl"}
-        }
+        volumes = [
+            (temp_subm_path, f"/autograder/submission/{nb_basename}"),
+            (results_path, "/autograder/results/results.pkl")
+        ]
         if pdfs:
-            volumes.update({f"{pdf_path}": {"bind": f"/autograder/submission/{nb_name}.pdf"}})
-        container = client.containers.run(image, "/autograder/run_autograder", volumes=volumes,
-                                          detach=True, tty=False)
+            volumes.append(pdf_path, f"/autograder/submission/{nb_name}.pdf")
```

Comment:
I think this should be `volumes.append((pdf_path, f"/autograder/submission/{nb_name}.pdf"))` so we're appending a tuple rather than 2 separate elements.

Reply:
Well you're right. My mistake.